iso/os-release

```diff
@@ -0,0 +1,8 @@
+NAME=minishift-b2d
+VERSION=1.0.0
```

Comment:
-1, I am not sure why you still use hard-coded version please have a look to https://github.com/minishift/minishift-centos-iso/blob/master/Makefile#L36 and use `envsubst` for that.